exporter/exporterhelper/queue_sender.go

```diff
@@ -123,14 +125,13 @@ func (qs *queueSender) onTemporaryFailure(logger *zap.Logger, req internal.Reque
 }
 
 // start is invoked during service startup.
-func (qs *queueSender) start(ctx context.Context, host component.Host, set exporter.CreateSettings) error {
+func (qs *queueSender) start(ctx context.Context, host component.Host) error {
```

Comment:
```suggestion
func (qs *queueSender) Start(ctx context.Context, host component.Host) error {
```

Other start/shutdown methods should be updated as well

Reply:
Haha :) Thanks, was debugging this.